app/src/main/cpp/skyline/services/base_service.h

```diff
@@ -3,6 +3,7 @@
 
 #pragma once
 
+#include <cxxabi.h>
```

Comment:
This isn't an STD include per-se, it should come under `<functional>`

Reply:
it doesn't